### ISSUE TITLE
[Campinas/SP] Desenvolvedor Data Science na inGaia

### ISSUE THREAD

**Issue by stefanobaldo** [backend-br/vagas#838]
## Descrição da vaga
Estamos à procura de um Desenvolvedor com foco em Data Science para juntar-se à nossa equipe de TI.

O que você precisa para se candidatar a esta vaga?

* Experiência em aplicações para análise de dados e machine learning;
* Perfil analítico orientado à indicadores de negócio;
* Geração de insights através de visualizações e cruzamento de informações;
* Aprofundamento de análises para feature engineering.

## Local
Av. José de Souza Campos, 1549 - Cambuí, Campinas - SP,

## Benefícios
* Plano de saúde
* Seguro de vida
* Vale Refeição
* Vale Transporte
* Mesa de Sinuca
* Lanches e festas de aniversário.
* Apoio Pass (Suporte Pessoal).

## Responsabilidades e atribuições:

* Apoiar equipe de planejamento com visão analítica e data driven;
* Cuidar dos modelos atuais, construir e validar novos modelos;
* Comunicar e comparar cenários e melhorias para o negócio através da utilização de técnicas de modelagem;
* Construção de modelos de Machine learning como: classificação, regressão e clusterização.

## Requisitos

**Obrigatórios:**

* Ferramentas e linguagens voltadas à análise e processamento de dados e machine learning como: Python/TensorFlow, R, Julia, Scala, Java, SAS, SQL, etc;
* Experiência com engenharia de dados e seleção de features.

**Altamente Desejável:**

* Experiência com modelos estatísticos e matemática;
* Experiência com Apache Beam, Spark, Flink, SAS Miner;
* Experiência com Keras, Scipy, etc;
* Experiência com Big Data (Hadoop ou serviços AWS/GCP);
* Experiência com plataformas de cloud (AWS/GCP/Azure);
* Experiência com dados não estruturados e possivelmente incompletos;
* Experiência com grandes volumes de dados;
* Experiência com otimização linear e não linear;
* Conhecimento de tópicos específicos de matemática e estatística: álgebra linear, correlação e causalidade, probabilidade, distribuição normal.

## Contratação

CLT

## Nossa empresa

* **Somos a inGaia:**  A maior empresa de tecnologia do setor imobiliário, com uma plataforma de soluções completa para os profissionais deste mercado. Ajudamos mais de 3 milhões de proprietários, compradores e imobiliárias a fecharem negócios.
* **Nós nascemos para construir um novo mercado imobiliário:** Com soluções que geram mais valor e ajudam os corretores de imóveis a trabalharem de forma mais profissional e eficiente.
* **Nosso time:** Performances individuais são sempre valorizadas, mas não toleramos individualismo e falta de ética. Trabalhamos juntos e somos positivos em todas as nossas atitudes, o mi-mi-mi não faz parte da nossa cultura.

## Como se candidatar

Por favor envie um email para [stefano.baldo@ingaia.com.br](mailto:stefano.baldo@ingaia.com.br) com seu CV anexado. Enviar no assunto: Tech Lead Back-End - GitHub

## Labels

* Alocado
* CLT
* Sênior
* Data Science
* Python
* Tensorflow

**Comment by reginaldojunior** [comment]
Issue fechada pelo bot. Motivo: Sem interações em um periodo de 3 meses.
